<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Nello Blaser - research</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Nello Blaser</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="teaching.html">Teaching</a>
</li>
<li>
  <a href="students.html">Students</a>
</li>
<li>
  <a href="news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://www.uib.no/personer/Nello.Blaser">
    <span class="fa fa-envelope fa"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/blasern">
    <span class="fa fa-github fa"></span>
     
  </a>
</li>
<li>
  <a href="https://arxiv.org/a/blaser_n_1.html">
    <span class="fa fa-scroll fas"></span>
     
  </a>
</li>
<li>
  <a href="https://orcid.org/0000-0001-9489-1657">
    <span class="fa fa-id-badge fa"></span>
     
  </a>
</li>
<li>
  <a href="https://dblp.org/pid/224/1086.html">
    <span class="fa fa-fort-awesome fa"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.com/citations?user=xkn66FYAAAAJ&amp;hl=no&amp;oi=ao">
    <span class="fa fa-graduation-cap fa"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<p><link rel="stylesheet" href="styles.css" type="text/css"></p>
<p>My current research is mainly focused on topological and geometric machine learning and its application in medicine. Previously I have worked on topics ranging from epidemiology, toxicology and geopyhsics to statistics and topological data analysis.</p>
<div id="projects" class="section level2">
<h2>Projects</h2>
<p>I am currently involved in the following funded research projects:</p>
<ul>
<li><a href="https://prosjektbanken.forskningsradet.no/en/project/FORISS/336988">Retail Fresh</a></li>
<li><a href="https://prosjektbanken.forskningsradet.no/en/project/FORISS/344236">DISTINGUISH</a></li>
<li><a href="https://data.snf.ch/grants/grant/235713">Differentiable Multi-Parameter Representation of Complex Data</a></li>
<li>Exploring Multimodality in Weather and Climate Prediction</li>
</ul>
<p>I have previously been involved in the following funded research projects:</p>
<ul>
<li><a href="https://prosjektbanken.forskningsradet.no/en/project/FORISS/274526">Parameterized Complexity for Practical Computing</a></li>
<li><a href="https://www.uib.no/en/dcod">dCod 1.0 - Decoding the Systems Toxicology of Atlantic Cod</a></li>
<li><a href="https://www.iedea-sa.org/">International epidemiology Databases to Evaluate AIDS Southern Africa collaboration (IeDEA-SA)</a></li>
</ul>
</div>
<div id="preprints" class="section level2">
<h2>Preprints</h2>
<p>Below you find a list of preprints.</p>
<p>[1] O. H. Gardaa and N. Blaser, “RotaTouille: Rotation equivariant deep learning for contours.” 2025, url: <a href="https://arxiv.org/abs/2508.16359" class="uri">https://arxiv.org/abs/2508.16359</a>.</p>
<p>[2] A. Grefsrud, T. Buanes, and N. Blaser, “Calibrated and uncertain? Evaluating uncertainty estimates in binary classification models.” 2025, url: <a href="https://arxiv.org/abs/2508.11460" class="uri">https://arxiv.org/abs/2508.11460</a>.</p>
<p>[3] S. Gavasso <em>et al.</em>, “High-dimensional immune profiling following autologous hematopoietic stem cell transplantation in relapsing-remitting multiple sclerosis,” 2025, doi: <a href="https://doi.org/10.1101/2025.07.01.662494">10.1101/2025.07.01.662494</a>.</p>
<p>[4] M. H. Hannisdal <em>et al.</em>, “Spatial-temporal recurrence patterns of grade 4 glioma using deep learning integrated mpMRI and molecular pathology: A multi-centre observational study,” 2025, doi: <a href="http://dx.doi.org/10.2139/ssrn.5177606" class="uri">http://dx.doi.org/10.2139/ssrn.5177606</a>.</p>
<p>[5] F. Ballerin, N. Blaser, and E. Grong, “SO(3)-equivariant neural networks for learning vector fields on spheres.” 2025, url: <a href="https://arxiv.org/abs/2503.09456" class="uri">https://arxiv.org/abs/2503.09456</a>.</p>
<p>[6] N. Blaser, M. Brun, O. H. Gardaa, and L. M. Salbu, “Core bifiltration.” 2024, url: <a href="https://arxiv.org/abs/2405.01214" class="uri">https://arxiv.org/abs/2405.01214</a>.</p>
<p>[7] N. Blaser, M. Brun, O. H. Gardaa, and L. M. Salbu, “Monoidal rips: Stable multiparameter filtrations of directed networks.” 2025, url: <a href="https://arxiv.org/abs/2503.14019" class="uri">https://arxiv.org/abs/2503.14019</a>.</p>
</div>
<div id="publications" class="section level2">
<h2>Publications</h2>
<p>Below you find a list of peer-reviewed publications.</p>
<p>[1] N. Blaser and E. R. Vågset, “Homology localization through the looking-glass of parameterized complexity theory,” <em>Journal of Applied and Computational Topology</em>, 2025, doi: <a href="https://doi.org/10.1007/s41468-025-00212-0">10.1007/s41468-025-00212-0</a>.</p>
<p>[2] B. Bergougnoux, N. Blaser, M. Fellows, P. Golovach, F. Rosamond, and E. Sam, “On the parameterized complexity of lineal topologies (depth-first spanning trees) with many or few leaves,” <em>Journal of Computer and System Sciences</em>, vol. 154, p. 103680, 2025, doi: <a href="https://doi.org/10.1016/j.jcss.2025.103680" class="uri">https://doi.org/10.1016/j.jcss.2025.103680</a>.</p>
<p>[3] M. Blørstad, B. Å. S. Lunde, and N. Blaser, “Stable update of regression trees,” in <em>Proceedings of the 3rd conference on lifelong learning agents</em>, 2025, vol. 274, pp. 641–651, url: <a href="https://proceedings.mlr.press/v274/blorstad25a.html" class="uri">https://proceedings.mlr.press/v274/blorstad25a.html</a>.</p>
<p>[4] N. Blaser, M. Brun, L. M. Salbu, and E. R. Vågset, “The parameterized complexity of finding minimum bounded chains,” <em>Computational Geometry</em>, vol. 122, p. 102102, 2024, doi: <a href="https://doi.org/10.1016/j.comgeo.2024.102102" class="uri">https://doi.org/10.1016/j.comgeo.2024.102102</a>.</p>
<p>[5] E. Sam, B. Bergougnoux, P. A. Golovach, and N. Blaser, “Kernelization for finding lineal topologies (depth-first spanning trees) with many or few leaves,” in <em>Fundamentals of computation theory</em>, 2023, pp. 392–405, doi: <a href="https://doi.org/10.1007/978-3-031-43587-4_28">10.1007/978-3-031-43587-4_28</a>.</p>
<p>[6] M. Eide <em>et al.</em>, “Integrative omics-analysis of lipid metabolism regulation by peroxisome proliferator-activated receptor a and b agonists in male atlantic cod (gadus morhua),” <em>Frontiers in Physiology</em>, vol. 14, p. 437, 2023, doi: <a href="https://doi.org/10.3389/fphys.2023.1129089">10.3389/fphys.2023.1129089</a>.</p>
<p>[7] B. Tislevoll <em>et al.</em>, “<span class="nocase">Early response evaluation by single cell signaling profiling in acute myeloid leukemia</span>,” <em>Nature Communications</em>, vol. 14, no. 1, p. 115, Jan. 2023, doi: <a href="https://doi.org/10.1038/s41467-022-35624-4">10.1038/s41467-022-35624-4</a>.</p>
<p>[8] S. Mayala <em>et al.</em>, “GUBS: Graph-based unsupervised brain segmentation in MRI images,” <em>Journal of Imaging</em>, vol. 8, no. 10, 2022, doi: <a href="https://doi.org/10.3390/jimaging8100262">10.3390/jimaging8100262</a>.</p>
<p>[9] N. Blaser and M. Brun, “Relative Persistent Homology,” <em>Discrete &amp; Computational Geometry</em>, 2022, doi: <a href="https://doi.org/10.1007/s00454-022-00421-9">10.1007/s00454-022-00421-9</a>.</p>
<p>[10] M. Black, N. Blaser, A. Nayyeri, and E. R. Vågset, “<span class="nocase">ETH-Tight Algorithms for Finding Surfaces in Simplicial Complexes of Bounded Treewidth</span>,” in <em>38th international symposium on computational geometry (SoCG 2022)</em>, 2022, vol. 224, pp. 17:1–17:16, doi: <a href="https://doi.org/10.4230/LIPIcs.SoCG.2022.17">10.4230/LIPIcs.SoCG.2022.17</a>.</p>
<p>[11] G. H. Bringeland, N. Blaser, K.-M. Myhr, C. A. Vedeler, and S. Gavasso, “Wearing-off symptoms during standard and extended natalizumab dosing intervals: Experiences from the COVID-19 pandemic,” <em>Journal of the Neurological Sciences</em>, p. 117622, 2021, doi: <a href="https://doi.org/10.1016/j.jns.2021.117622">10.1016/j.jns.2021.117622</a>.</p>
<p>[12] N. Galmiche, H. Hauser, T. Spengler, C. Spensberger, M. Brun, and N. Blaser, “<span class="nocase">Revealing Multimodality in Ensemble Weather Prediction</span>,” in <em>Machine learning methods in visualisation for big data</em>, 2021, doi: <a href="https://doi.org/10.2312/mlvis.20211073">10.2312/mlvis.20211073</a>.</p>
<p>[13] K. Gundersen, A. Oleynik, N. Blaser, and G. Alendal, “Semi-conditional variational auto-encoder for flow reconstruction and uncertainty quantification from limited observations,” <em>Physics of Fluids</em>, vol. 33, no. 1, p. 017119, 2021, doi: <a href="https://doi.org/10.1063/5.0025779">10.1063/5.0025779</a>.</p>
<p>[14] N. Blaser and M. Aupetit, “Research directions to validate topological models of multi-dimensional data,” 2020, url: <a href="https://openreview.net/forum?id=3iVbgfPMZtU" class="uri">https://openreview.net/forum?id=3iVbgfPMZtU</a>.</p>
<p>[15] K. Gundersen, G. Alendal, A. Oleynik, and N. Blaser, “Binary time series classification with bayesian convolutional neural networks when monitoring for marine gas discharges,” <em>Algorithms</em>, vol. 13, p. 145, Jun. 2020, doi: <a href="https://doi.org/10.3390/a13060145">10.3390/a13060145</a>.</p>
<p>[16] N. Blaser and M. Brun, “Relative Persistent Homology,” in <em>36th international symposium on computational geometry (SoCG 2020)</em>, 2020, vol. 164, pp. 18:1–18:10, doi: <a href="https://doi.org/10.4230/LIPIcs.SoCG.2020.18">10.4230/LIPIcs.SoCG.2020.18</a>.</p>
<p>[17] A. Oleynik, M. I. García-Ibáñez, N. Blaser, A. Omar, and G. Alendal, “Optimal sensors placement for detecting CO2 discharges from unknown locations on the seafloor,” <em>International Journal of Greenhouse Gas Control</em>, vol. 95, p. 102951, 2020, doi: <a href="https://doi.org/10.1016/j.ijggc.2019.102951">10.1016/j.ijggc.2019.102951</a>.</p>
<p>[18] G. H. Bringeland, N. Blaser, K.-M. Myhr, C. A. Vedeler, and S. Gavasso, “Wearing-off at the end of natalizumab dosing intervals is associated with low receptor occupancy,” <em>Neurology - Neuroimmunology Neuroinflammation</em>, vol. 7, no. 3, 2020, doi: <a href="https://doi.org/10.1212/NXI.0000000000000678">10.1212/NXI.0000000000000678</a>.</p>
<p>[19] N. Blaser and M. Brun, “Sparse nerves in practice,” in <em>Machine learning and knowledge extraction</em>, 2019, pp. 272–284, doi: <a href="https://doi.org/10.1007/978-3-030-29726-8_17">10.1007/978-3-030-29726-8_17</a>.</p>
<p>[20] L. Bader <em>et al.</em>, “Candidate markers for stratification and classification in rheumatoid arthritis,” <em>Frontiers in Immunology</em>, vol. 10, p. 1488, 2019, doi: <a href="https://doi.org/10.3389/fimmu.2019.01488">10.3389/fimmu.2019.01488</a>.</p>
<p>[21] M. Brun and N. Blaser, “Sparse dowker nerves,” <em>Journal of Applied and Computational Topology</em>, pp. 1–28, 2019, doi: <a href="https://doi.org/10.1007/s41468-019-00028-9">10.1007/s41468-019-00028-9</a>.</p>
<p>[22] K. Dale <em>et al.</em>, “Contaminant accumulation and biological responses in atlantic cod (gadus morhua) caged at a capped waste disposal site in kollevåg, western norway,” <em>Marine Environmental Research</em>, 2019, doi: <a href="https://doi.org/10.1016/j.marenvres.2019.02.003">10.1016/j.marenvres.2019.02.003</a>.</p>
<p>[23] G. H. Bringeland <em>et al.</em>, “Optimization of receptor occupancy assays in mass cytometry: Standardization across channels with QSC beads,” <em>Cytometry Part A</em>, 2019, doi: <a href="https://doi.org/10.1002/cyto.a.23723">10.1002/cyto.a.23723</a>.</p>
<p>[24] K. E. M. Ahmed <em>et al.</em>, “An LC-MS/MS approach to reveal the effects of POPs and endocrine disruptors on the steroidogenesis of the human H295R adrenocortical cell line,” <em>Chemosphere</em>, 2019, doi: <a href="https://doi.org/10.1016/j.chemosphere.2018.11.057">10.1016/j.chemosphere.2018.11.057</a>.</p>
<p>[25] H. G. Frøysa, S. Fallahi, and N. Blaser, “Evaluating model reduction under parameter uncertainty,” <em>BMC Systems Biology</em>, vol. 12, no. 1, p. 79, 2018, doi: <a href="https://doi.org/10.1186/s12918-018-0602-x">10.1186/s12918-018-0602-x</a>.</p>
<p>[26] N. Blaser and M. Brun, “Divisive cover,” <em>Mathematics in Computer Science</em>, vol. 13, no. 1, pp. 21–29, 2019, doi: <a href="https://doi.org/10.1007/s11786-018-0352-6">10.1007/s11786-018-0352-6</a>.</p>
<p>[27] F. Yadetie <em>et al.</em>, “RNA-seq analysis of transcriptome responses in atlantic cod (gadus morhua) precision-cut liver slices exposed to benzo[a]pyrene and 17–ethynylestradiol,” <em>Aquatic Toxicology</em>, vol. 201, pp. 174–186, 2018, doi: <a href="https://doi.org/10.1016/j.aquatox.2018.06.003">10.1016/j.aquatox.2018.06.003</a>.</p>
<p>[28] J. Estill <em>et al.</em>, “The effect of monitoring viral load and tracing patients lost to follow-up on the course of the HIV epidemic in malawi: A mathematical model,” <em>Open Forum Infectious Diseases</em>, vol. 5, no. 5, p. ofy092, 2018, doi: <a href="https://doi.org/10.1093/ofid/ofy092">10.1093/ofid/ofy092</a>.</p>
<p>[29] N. Blaser <em>et al.</em>, “Impact of screening and ART on anal cancer incidence in HIV-positive men who have sex with men: Mathematical modeling study,” <em>AIDS</em>, 2017, doi: <a href="https://doi.org/10.1097/QAD.0000000000001546">10.1097/QAD.0000000000001546</a>.</p>
<p>[30] J. Estill <em>et al.</em>, “Estimating the need of second-line antiretroviral therapy in sub-Saharan Africa up to 2030: A mathematical model,” <em>The Lancet HIV</em>, vol. 3, no. 3, pp. e132–e139, 2016, doi: <a href="https://doi.org/10.1016/S2352-3018(16)00016-3">10.1016/S2352-3018(16)00016-3</a>.</p>
<p>[31] N. Blaser <em>et al.</em>, “Tuberculosis in cape town: An age-structured transmission model,” <em>Epidemics</em>, vol. 14, pp. 54–61, 2016, doi: <a href="https://doi.org/10.1016/j.epidem.2015.10.001">10.1016/j.epidem.2015.10.001</a>.</p>
<p>[32] O. Keiser <em>et al.</em>, “Growth in Virologically Suppressed HIV Positive Children on Antiretroviral Therapy: Individual and Population-Level References,” <em>The Pediatric Infectious Disease Journal</em>, 2015, doi: <a href="https://doi.org/10.1097/INF.0000000000000801">10.1097/INF.0000000000000801</a>.</p>
<p>[33] J. Estill, L. Salazar-Vizcaya, N. Blaser, M. Egger, and O. Keiser, “The Cost-Effectiveness of Monitoring Strategies for Antiretroviral Therapy of HIV Infected Patients in Resource-Limited Settings: Software Tool,” <em>PLoS ONE</em>, vol. 10, no. 3, p. e0119299, 2015, doi: <a href="https://doi.org/10.1371/journal.pone.0119299">10.1371/journal.pone.0119299</a>.</p>
<p>[34] N. Blaser <em>et al.</em>, “<span class="nocase">gems</span>: An R package for simulating from disease progression models,” <em>Journal of Statistical Software</em>, vol. 64, no. 10, pp. 1–22, 2015, doi: <a href="https://doi.org/10.18637/jss.v064.i10">10.18637/jss.v064.i10</a>.</p>
<p>[35] L. Salazar-Vizcaya <em>et al.</em>, “Viral load versus CD4<sup>+</sup> monitoring and 5-year outcomes of antiretroviral therapy in HIV-positive children in Southern Africa: A cohort-based modelling study,” <em>AIDS</em>, vol. 28, no. 16, pp. 2451–2460, 2014, doi: <a href="https://doi.org/10.1097/QAD.0000000000000446">10.1097/QAD.0000000000000446</a>.</p>
<p>[36] M. Petersen, J. Schwab, S. Gruber, N. Blaser, M. Schomaker, and M. van der Laan, “Targeted maximum likelihood estimation for dynamic and static longitudinal marginal structural working models,” <em>Journal of Causal Inference</em>, vol. 2, no. 2, pp. 147–185, 2014, doi: <a href="https://doi.org/10.1515/jci-2013-0007">10.1515/jci-2013-0007</a>.</p>
<p>[37] N. Blaser <em>et al.</em>, “Impact of viral load and the duration of primary infection on HIV transmission: Systematic review and meta-analysis,” <em>AIDS</em>, vol. 28, no. 7, pp. 1021–1029, 2014, doi: <a href="https://doi.org/10.1097/QAD.0000000000000135">10.1097/QAD.0000000000000135</a>.</p>
<p>[38] J. Estill <em>et al.</em>, “Tracing of patients lost to follow-up and HIV transmission: Mathematical modeling study based on 2 large ART programs in Malawi,” <em>Journal of acquired immune deficiency syndromes</em>, vol. 65, no. 5, pp. e179–186, 2014, doi: <a href="https://doi.org/10.1097/QAI.0000000000000075">10.1097/QAI.0000000000000075</a>.</p>
<p>[39] D. Keebler <em>et al.</em>, “Cost-effectiveness of different strategies to monitor adults on antiretroviral treatment: A combined analysis of three mathematical models,” <em>The Lancet Global Health</em>, vol. 2, no. 1, pp. e35–e43, 2014, doi: <a href="https://doi.org/10.1016/S2214-109X(13)70048-2">10.1016/S2214-109X(13)70048-2</a>.</p>
<p>[40] J. Estill <em>et al.</em>, “Cost-effectiveness of point-of-care viral load monitoring of antiretroviral therapy in resource-limited settings: Mathematical modelling study,” <em>AIDS</em>, vol. 27, no. 9, pp. 1483–1492, 2013, doi: <a href="https://doi.org/10.1097/QAD.0b013e328360a4e5">10.1097/QAD.0b013e328360a4e5</a>.</p>
<p>[41] C. Wettstein <em>et al.</em>, “Missed opportunities to prevent mother-to-child-transmission: Systematic review and meta-analysis,” <em>AIDS</em>, vol. 26, no. 18, pp. 2361–2373, 2012, doi: <a href="https://doi.org/10.1097/QAD.0b013e328359ab0c">10.1097/QAD.0b013e328359ab0c</a>.</p>
</div>

<script type="text/javascript" src="links.js"></script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
